compiler/rustc_attr_parsing/src/context.rs

```diff
@@ -121,6 +121,7 @@ attribute_parsers!(
         // tidy-alphabetical-start
         Combine<AllowConstFnUnstableParser>,
         Combine<AllowInternalUnstableParser>,
+        Combine<CfgParser>,
```

Comment:
Is this even needed since we don't parse the attr late?

Reply:
Removed